src/main/java/org/sciborgs1155/robot/shooter/Shooter.java

```diff
@@ -0,0 +1,72 @@
+package org.sciborgs1155.robot.shooter;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.util.function.DoubleSupplier;
+import monologue.Logged;
+import monologue.Monologue.LogBoth;
+import org.sciborgs1155.robot.Robot;
+
+public class Shooter extends SubsystemBase implements Logged, AutoCloseable {
+  private final FlywheelIO flywheel;
+
+  @LogBoth
+  private final PIDController pidController =
+      new PIDController(ShooterConstants.kp, ShooterConstants.ki, ShooterConstants.kd);
+      
+
+  private final SimpleMotorFeedforward feedForward =
+      new SimpleMotorFeedforward(
+          ShooterConstants.kSVolts, ShooterConstants.kVVoltSecondsPerRotation);
+
+  @LogBoth public double target;
+
+  @LogBoth
+  public boolean isAtGoal() {
+    return pidController.atSetpoint();
+  }
+
+  public Shooter(FlywheelIO flywheelIOtype) {
```

Comment:
If you do it like this, call it flywheel, not flywheelIOtype (that name doesn't really make sense)

Alternatively, you can have no create() and at the top where you declare the flywheel, you can instead have:
private final FlywheelIO flywheel = Robot.isReal() ? new RealFlywheel() : new SimFlywheel();

I think the second way is a little cleaner